test/functional/apps/discover/_runtime_fields_editor.ts

```diff
@@ -43,18 +44,20 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     });
 
     after(async () => {
-      await kibanaServer.uiSettings.replace({ 'discover:searchFieldsFromSource': true });
+      await esArchiver.unload('logstash_functional');
     });
 
     it('allows adding custom label to existing fields', async function () {
+      const uuid = uuidv4();
+      const customLabel = `megabytes${uuid}`;
       await PageObjects.discover.clickFieldListItemAdd('bytes');
```

Comment:
This could be the reason for the failure, when you add a field to the table, the index pattern is updated, the popularity counter of the field is increased. So maybe, since in functional testing it happens a bit faster than in our human world, unless you're a robot, this could lead to a conflict, of those 2 updates? this could be resolved be adding the field to the table after assigning the custom label, or just to check if the field in the sidebar has a custom label